packages/Popover/src/popover.scss

```diff
@@ -14,19 +18,45 @@
   --ids-popover-text: #{tokens.$grey-600};
   --ids-popover-background: #{tokens.$samoyed};
   --ids-popover-shadow: #{tokens.$shadow-12};
+  --ids-popover-title-color: #{tokens.$grey-800};
+  --ids-popover-title-font-weight: #{tokens.$font-weight-semi-bold};
   --ids-popover-title-padding: #{tokens.$space-5};
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-popover-font-family: var(--hop-body-sm-font-family);
+  --ids-popover-font-size: var(--hop-body-sm-font-size);
+  --ids-popover-padding: var(--hop-space-inset-md);
+  --ids-popover-border: 0.063rem solid var(--hop-neutral-border-weak);
```

Comment:
0.063rem is an odd number

Reply:
the convertor I was using rounds them. It's supposed to be 1px but in rem

Reply:
0.0625rem is 1px